project.md

```diff
@@ -58,11 +96,21 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think this project would skew more towards systems. If I had to say, probably 70% systems and 30% language design. There are opportunities to work on the language aspect of it however. I would like to implement parser combinators such that users could either use natural language when describing an event or type only required syntax so as to make making a course calendar very quick. However, the DSL’s being made for this project largely focus on compatibility and exporting to ICS, and then also being able to parse and display them in a preview. I think most of the time spent will be on implementing ICS export/parsing ICS for the second DSL, which should mostly be systems focused.
+
+
 ### Scope
 
 _How big an idea is this? How ambitious is this project?_
 
+
+I think the project isn’t too big of an idea. I think others have worked on similar ideas and there already is an ICS standard. Therefore there should be libraries I can utilize and I can use much of the knowledge from previous HW assignments in this project. However, I think working on two DSL’s might prove to be difficult and might be time consuming. I think the project should not be too ambitious because there is already much documentation on ICS and different calendar standards, but it might be difficult to fit everything I want to do into 5 weeks should I run into any roadblocks.
+
+
 ### Benefits and drawbacks
 
 _Why might this be a good idea for a project? Why might this not be a good idea
 project?_
+
+I think this is a fantastic idea for a project because it’s something I’m personally interested in. I also think it’s a good idea because it could help others stay organized, and it could help myself stay organized. It’s also something that could be used to explore different tradeoffs for host languages and DSL design techniques.
+I think it might not be a good idea because it will largely be focused on systems instead of language design. This might give me a limited understanding of DSL’s, and might instead leave me with a better understanding of the .ICS file type. However, professor Weidermann has mentioned it’s a great idea and that it’s viable, so that shouldn’t be too big of a problem.
```

Comment:
I think it's doable! There are definitely choices that you could make that would skew it systems vs language design timewise, but since you've talked to Prof Ben and you are aware, I think you'll be fine.